testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -520,4 +524,41 @@ public void testSelectInformationSchemaColumns()
                         "('tables'), " +
                         "('views')");
     }
+
+    @Test
+    public void testRenameTableAcrossSchema()
+    {
+        if (!hasBehavior(SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS)) {
+            if (!hasBehavior(SUPPORTS_RENAME_TABLE)) {
+                return;
```

Comment:
add a comment why

Reply:
I added a `SkipException` with a message, hope that's enough.